source/_layouts/master.blade.php

```diff
@@ -181,6 +181,22 @@
                                         <path d="M28.784 3.226l-10.508 24.556-3.729-10.766-10.756-3.251zM31.389 0.024c-0.191 0-0.422 0.054-0.691 0.167l-29.833 12.659c-1.075 0.456-1.143 1.335-0.151 1.952l12.353 3.862 4.043 12.602c0.29 0.474 0.64 0.71 0.977 0.71 0.373 0 0.728-0.286 0.97-0.852l12.759-29.804c0.346-0.809 0.149-1.296-0.426-1.296z"></path>
                                     </svg>
                                 </a>
+                            </li>
+                            
+                                 <li>
+                                <a href="@yield('main.discord')" target="https://discord.gg/aEVdGMyRt4" rel="noopener noreferrer" title="Discord">
```

Comment:
Are you sure it's correct? Where did the template get the key `main.discord` from? 
Is there a `target` attribute with the parameter that you pass?

Reply:
You can see how we put different links for the Russian and English versions of the website.
https://github.com/orchidsoftware/orchid.software/blob/3185a0b0b0b84f8bd3f4eee8eee4afb9e86ec75a/source/_lang/en.blade.php#L87

Reply:
I have fixed the URL. All the channel invites will lead to the same server.